tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -191,3 +191,11 @@ Login Via Button
    ...  Keyword.
    Click Element  xpath://a[@id='login']
    Wait Until Page Contains  Log in with
+
+Access Spawner API
+  [Arguments]  ${Target}
+  Click Element  xpath://*[@id="thenavbar"]/ul[1]/li[4]/a
+  Click Element  xpath://*[@id="thenavbar"]/ul[1]/li[4]/ul/li/a
```

Comment:
it might make sense to add a small check here that you are in the correct page before trying to click the elements, i.e. `Page Should Contain  ...`